councilmatic_core/haystack_indexes.py

```diff
@@ -58,16 +56,20 @@ def prepare_full_text(self, obj):
         return clean_html(obj.full_text)
 
     def prepare_last_action_date(self, obj):
-        from datetime import datetime, timedelta
-        if not obj.last_action_date:
-            return datetime.now().replace(tzinfo=app_timezone) - timedelta(days=36500)
-        return obj.last_action_date
+        from datetime import timedelta
+        last_date = obj.last_action_date
+        if not last_date:
+            last_date = timezone.now() - timedelta(36500)
+
+        if last_date:
+            last_date = last_date - last_date.utcoffset()
```

Comment:
i'll be honest with you, i have no idea

Reply:
i think that an easier question to answer is: how should the index handle bills without a last action date? 

it seems like this is placing them way, way in the past. is this for the legislative session filter? if so, then it would have the effect of not returning bills with this funny date for any of the current legislative sessions. i think that we could achieve this by simply returning `None` if there is no `last_action_date`, no?

Reply:
i think that this has to do with https://github.com/datamade/chi-councilmatic/blob/master/chicago/search_indexes.py#L20-L26

but we should just handle that kind of changes in that code, and we override this method anyway. I think we can drop this method altogether.

Reply:
done! rebuilt the chicago index to make sure it didn't break anything, and it looks ok to me!